features/tasks/deleteTask.feature

```diff
@@ -28,10 +28,10 @@ Feature: Set of tests to delete tasks
 		And <userRole> sets task body '<taskBody>' in task form
 		And <userRole> clicks Add-task-submit button
 		When <userRole> goes to tasks page
-		Then <userRole> should see that task with name '<taskName>' is visible on the list
+		Then <userRole> should see task with name '<taskName>' visible
```

Comment:
I think we can also get rid of the 'visible' at the end of the step. Otherwise it sounds rather strange. Same for the step in line 34.